misc/pytorch_toolkit/chest_xray_screening/chest_xray_screening/utils/download_weights.py

```diff
@@ -0,0 +1,11 @@
+import os
+from google_drive_downloader import GoogleDriveDownloader as gdd
+
+def download_checkpoint():
+    os.makedirs('model_weights')
+    gdd.download_file_from_google_drive(file_id='13KtMdL-RJoTXGRZ8WIdcHkf0XuX9bMMN',
+                                    dest_path='model_weights/chest_xray_screening.pth',
```

Comment:
I think, it would be better to leave this function as a wrapper for `gdd.download_file_from_google_drive`. Add parameters for destination and file id, store these parameters in the config file of the model.